src/taipy/core/taipy.py

```diff
@@ -76,6 +78,8 @@ def submit(
         timeout (Union[float, int]): The optional maximum number of seconds to wait for the jobs to be finished before returning.
 
     """
+    if _SchedulerFactory._dispatcher is None:
+        _warn_no_core_service()
```

Comment:
Minor remark: I would put that small piece of code in a decorator so it is easy to re-use it in other entry points if needed.